config/netlink.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import (
+	"time"
+
+	"github.com/bio-routing/bio-rd/routingtable/filter"
+)
+
+// Constants for default routing tables in the Linux Kernel
+const (
+	RtLocal   int = 255 // according to man ip-route: 255 is reserved fro built-in use
+	RtMain    int = 254 // This is the default table where routes are inserted
+	RtDefault int = 253 // according to man ip-route: 253 is reserved fro built-in use
+	RtUnspec  int = 0   // according to man ip-route: 0 is reserved fro built-in use
```

Comment:
s/fro/for

Reply:
fixed. thanks. looks like an c&p issue xD